app/src/main/res/values/strings.xml

```diff
@@ -34,7 +34,7 @@
     <string name="appmananger_app_delete">Delete</string>
     <string name="appmananger_app_delete_cache">Delete and remove from cache</string>
     <string name="appmananger_app_reinstall">Reinstall</string>
-    <string name="appmanager_app_openinstore">Search in Pebble Appstore</string>
+    <string name="appmanager_app_openinstore">Search in Pebble appstore</string>
     <string name="appmanager_health_activate">Activate</string>
```

Comment:
No idea about upper and lower case in English

Reply:
As per https://github.com/pebble-dev/wiki/wiki/Appstore